### PR TITLE
Us01 3 4

### PR DESCRIPTION
**Pull request by LudovicRAVINET**
add index links and move contents under navbar to prepare friday demo

### CODE REVIEW

File: src/View/Home/index.html.twig [hunk 1]
```diff
@@ -30,7 +30,7 @@
 ---------------------------------------------------------- -->
 
     
-    <button id="show_popup">Show popup</button>
+    <!--<button id="show_popup">Show popup</button>-->
```
Comment (aldaitzwild) [inline src/View/Home/index.html.twig:33]: No dead code !

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/View/Home/index.html.twig (hunk 1 above)
Comment (LudovicRAVINET) [reply src/View/Home/index.html.twig]: ok

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
Conflict with dev :

Conflicting files
src/View/SoughtProfile/soughtProfile.html.twig

File: src/View/SoughtProfile/soughtProfile.html.twig [hunk 2]
```diff
@@ -3,6 +3,7 @@
 {% block title %}Affichage des profils{% endblock %}
 
 {% block content %}
+<div class="cadre">
```
Comment (aldaitzwild) [inline src/View/SoughtProfile/soughtProfile.html.twig:6]: This div is not closed

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/View/SoughtProfile/soughtProfile.html.twig (hunk 2 above)
Comment (LudovicRAVINET) [reply src/View/SoughtProfile/soughtProfile.html.twig:6]: ok I forgot it :(

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**